script/templates/skeleton_version.h

```diff
@@ -0,0 +1,11 @@
+#ifndef %1$s_SKELETON_VERSION_H
+#define %1$s_SKELETON_VERSION_H
+
+#include <string>
+
+namespace %1$s
+{
+static const std::string version = "2.0.0";
+}
+
+#endif
```

Comment:
Is this the version of the BioCro framework that's included with the skeleton library or is it something else?